app/src/main/java/com/codepath/michfeng/songswiper/connectors/RunnableRecs.java

```diff
@@ -21,16 +25,21 @@ public class RunnableRecs implements Runnable {
     RecommendationCollection recs;
     SpotifyApi spotifyApi;
     private volatile boolean finish;
+    Context context;
 
     private static final String TAG = "RunnableRecs";
 
-    public RunnableRecs(SpotifyApi api) {
+    public RunnableRecs(SpotifyApi api, Context context) {
         this.spotifyApi = api;
+        this.context = context;
     }
 
     @Override
     public void run() {
         try  {
+            Log.i(TAG, "starting loading screen");
```

Comment:
I suggest capitalizing logs, since it's more readable for other devs who may share the same logs as you.